extension/app/src/components/conversation/ConversationViewer.tsx

```diff
@@ -106,15 +108,15 @@ export function ConversationViewer({
  * together.
  *
  * Example:
- * Input [[content_fragment, content_fragment], [user_message], [agent_message, agent_message]]
- * Output: [[user_message with content_fragment[]], [agent_message, agent_message]]
+ * Input [content_fragment, content_fragment, user_message, agent_message, agent_message, user_message, agent_message]
```

Comment:
_I think we could rewrite the doc to clarify, suggestion of wording but feel free to use something else:_ 

We're grouping messages by question/answer, meaning: user message + potential content fragments posted with the user message + one or multiple agent messages depending on the number of mentions in the user message. 

That means we want this: 
- Input [content_fragment, content_fragment, user_message, agent_message, agent_message, user_message, agent_message]
- Output [[user_message with content_fragment[], agent_message, agent_message], [user_message, agent_message ]]